packages/cypress/cypress/support/commands.js

```diff
@@ -51,7 +51,7 @@ Cypress.Commands.add('login', () => {
         url: `${API_BASE_URL}/session`,
         body: {
           sessionLocation: DEFAULT_LOCATION_UUID,
-          locale: "en_GB",
+          locale: "en",
```

Comment:
brill catch @jayasanka-sack!
@rbuisson [made a Clean up on the list of supported locales ](https://github.com/openmrs/openmrs-distro-referenceapplication/commit/9d1750d97cf788dc9361e78a0a56e278be98c4da)